libs/designer-ui/src/lib/copilotGetStarted/components/flowPreview.tsx

```diff
@@ -0,0 +1,306 @@
+import Constants from '../../constants';
+import { FontIcon, Icon } from '@fluentui/react/lib/Icon';
+import { css } from '@fluentui/react/lib/Utilities';
+import React from 'react';
+import type { OperationInfo} from '../../chatbot/components/flowDiffPreview';
+import { OperationItem } from '../../chatbot/components/flowDiffPreview';
+import { useIntl } from 'react-intl';
+
+export interface IFlowDefinition {
+    $schema?: string;
+    contentVersion?: string;
+    description?: string;
+    dependencies?: [] | null;
+
+    actions?: any; // eslint-disable-line @typescript-eslint/no-explicit-any
+    metadata?: any; // eslint-disable-line @typescript-eslint/no-explicit-any
+    outputs?: any; // eslint-disable-line @typescript-eslint/no-explicit-any
+    parameters?: any; // eslint-disable-line @typescript-eslint/no-explicit-any
+    triggers?: any; // eslint-disable-line @typescript-eslint/no-explicit-any
+}
+
+interface FlowPreviewProps {
+    definition: IFlowDefinition;
+    /* TODO: Add connection references, etc */
+    dataAutomationId?: string;
+}
+
+export function toOperationInfo(operationName: string, operation: OperationContract): OperationInfo {
```

Comment:
We can move these functions out into a helper file 

Also tbd but we'll either rename OperationInfo or try to maintain same data types as Designer

Reply:
sounds good!